src/Promotions/InPluginUpsells/resources/js/components/FreeAddOnTab.js

```diff
@@ -0,0 +1,102 @@
+import {useRef} from 'react';
+import cx from 'classnames';
+
+import GreenButton from '@givewp/promotions/components/GreenButton';
+import useFreeAddonSubscription from '@givewp/promotions/hooks/useFreeAddonSubscription';
+
+import styles from './FreeAddOnTab.module.css';
+import {Hero} from './Hero';
+import {Card} from './Card';
+import {transformStrong} from '../utils';
+
+const highlights = [
+    {
+        icon: 'https://givewp.com/downloads/upsells/images/time-icon.svg',
+        text: 'Flexible Delivery Times',
```

Comment:
@JasonTheAdams can we make these strings translatable?

Reply:
This is just leftover mock data. I've removed it.